PluginCore/PluginCore/Resources/en_US.resx

```diff
@@ -6189,4 +6189,8 @@ AND - All conditions are met.</value>
     <value>Keep result panels locked by default.</value>
     <comment>Added after 5.2.0</comment>
   </data>
+  <data name="HaXeContext.Info.UnusedCase" xml:space="preserve">
+    <value>Unreachable case</value>
```

Comment:
It seems like for 2 and 3 (`CompilerError` and `RemovableCode`), `res.args` and `res.args.description` values are used for the descriptions rather than hardcoded texts.